src/aws/cloudformation.coffee

```diff
@@ -118,4 +118,12 @@ module.exports = async (env, config) ->
     postDelete = async -> yield src.destroy()
 
 
-    {publish, delete: destroy, publishWait, deleteWait, postPublish, postDelete}
+    {
```

Comment:
i guess i understand the use of this, but ideally this is done via discovery, no?